commit 2923c7f93a2bca22a9a8caf735e2fab35f3aff49 (refs/changes/07/2407/2)
Author: brittany <brittany.wald@joyent.com>
Date:   2017-09-14T00:38:53+00:00 (2 years, 1 month ago)
    
    MANTA-3279 node-fash loads the whole leveldb hash ring into memory

diff --git a/package.json b/package.json
index 6df79ca..5fd45f4 100644
--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
         "cmdln": "3.2.1",
         "cmdutil": "1.0.0",
         "extsprintf": "1.3.0",
-        "fash": "2.3.1",
+        "fash": "2.4.1",
         "forkexec": "0.1.0",
         "hogan.js": "2.0.0",
         "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#65bba66818",
